src/utils/CtypeUtils/CtypeUtils.ts

```diff
@@ -90,19 +108,12 @@ export const getCTypeInputModel = (ctype: sdk.CType): ICTypeInput => {
  * @returns {any} The claim input model
  */
 export const getClaimInputModel = (
-  ctype: sdk.ICType,
+  ctype: sdk.CType,
```

Comment:
Why the change from the Interface to the class?

Reply:
this had to be a mistake, I reverted it.